tests/test_accessibility.py

```diff
@@ -0,0 +1,54 @@
+"""
+This is a test file to run accessibility test on
+    1. Selenium tutorial main page
+    2. Selenium tutorial redirect page
+    3. Selenium tutorial contact page
+    #While running the test for first time, use --snapshot-update to create a snapshot folder
```

Comment:
Can you create an issue to:
- Check if `snapshots` are available in a dir
- Run `snapshot-update` if the snapshots are not available

We can work on this later, for now running the test with `--snapshot-update` for the first time can be recommended.

Reply:
Sure.

Reply:
@ajitava-git Please let me know once you create the issue, I can approve this PR & we can go ahead with merging it.

Reply:
Done @shivahari